### PR TITLE
Development update dashboard

### PR DESCRIPTION
**Pull request by 0oM4R**
in this pr 
- update some images, 
- update create profile 
- fix broken link to create a profile 
### Related issue 
- #51

### CODE REVIEW

File: src/getstarted/TF_Dashboard/img/dashboard_8.png
Comment (Mik-TF) [inline src/getstarted/TF_Dashboard/img/dashboard_8.png:1]: There's a typo in : Your seed phrase is showen (sic) here.

"shown"

File: src/getstarted/TF_Dashboard/img/dashboard_5.png
Comment (Mik-TF) [inline src/getstarted/TF_Dashboard/img/dashboard_5.png:1]: With this picture, we do not "see" that you have to click on the Extensions button.

File: src/getstarted/TF_Dashboard/img/dashboard_9.png
Comment (Mik-TF) [inline src/getstarted/TF_Dashboard/img/dashboard_9.png:1]: With the new picture, we lose the steps required (choose a name, choose a password, etc.)

File: src/getstarted/TF_Dashboard/TF_Dashboard.md [hunk 1]
```diff
@@ -1,16 +1,3 @@
-<h1> Threefold Dashboard: Create an Account and Transfer TFT</h1>
```
Comment (Mik-TF) [inline src/getstarted/TF_Dashboard/TF_Dashboard.md:1]: Why do we remove the Title, the table of contents and the embedded video tutorial?

I do not understand the need to "change" the section by removing the work that was done.

**Comment by Mik-TF** [comment]
The [related issue](https://github.com/threefoldtech/info_grid/issues/51) asked for a small change of one picture to show that we need to create a relay instead of the twin:
"The steps for creating a twin should be updated, to replace the twin ip creation part with selecting the relay."

Do we just change this picture or we change the whole file?

Also, I can redo the section of the video with the relay part. I don't see why we would remove the video.
That being said, the relay change is not even yet on main net:

test_10


I propose to let the file as it was before, and to adjust with the relay picture and change the small video portion when this is available for main net.

File: src/getstarted/TF_Dashboard/img/dashboard_relay.png
Comment (Mik-TF) [inline src/getstarted/TF_Dashboard/img/dashboard_relay.png:1]: This picture is with .dev.grid.tf (dev net).
This might mislead users as this guide clearly shows main net dashboard links.

We should wait until the changes are on main net.

**Comment by mohamedamer453** [comment]
> 
> test_10
> 
> @mohamedamer453 I propose to let the file as it was before, and to adjust with the relay picture and change the small video portion when this is available for main net. (It is available for dev and test nets for now.)

Indeed, i totally agree with you replacing the mentioned image and adjusting other info about the twin ip/relay if needed was the main purpose of this issue, and i also totally agree about the video part if the section of the twin ip could be edited and replaced with the relay instead it would be much better than removing the whole video, since the video could be really helpful for new users. (and of course these changes should remain on the staging server and be released with the 3.9.0 mainnet release)

**Comment by Mik-TF** [comment]
OK. Good.

For now, the link would still be relay.dev.grid.tf, as the relay isn't yet on main net.

Once 3.9.0 is out, I can do very quickly the change in the video and the picture, as I will be able to record the proper relay address (relay.grid.tf) on screen.

Perhaps the simplest thing to do, is to close this pull request, as it won't be accepted, and also we can add your [original issue](https://github.com/threefoldtech/info_grid/issues/51) for the 3.9.0 release so we work on it when it's optimal.

Let me know what you think @mohamedamer453

File: src/getstarted/TF_Dashboard/TF_Dashboard.md (hunk 1 above)
Comment (0oM4R) [reply src/getstarted/TF_Dashboard/TF_Dashboard.md:1]: image

this is my suggestion, and we will add the video before the table of contents

File: src/getstarted/TF_Dashboard/img/dashboard_5.png
Comment (0oM4R) [reply src/getstarted/TF_Dashboard/img/dashboard_5.png:1]: Is this really needed, the user has three filed with their description :
`A DESCRIPTIVE NAME FOR YOUR ACCOUNT` ,  `A NEW PASSWORD FOR THIS ACCOUNT`, and `REPEAT PASSWORD FOR VERIFICATION`
in my opinion, there is no need to add more description

**Comment by 0oM4R** [comment]
> The [related issue](https://github.com/threefoldtech/info_grid/issues/51) asked for a small change of one picture to show that we need to create a relay instead of the twin: "The steps for creating a twin should be updated, to replace the twin ip creation part with selecting the relay."
> 
> Do we just change this picture or we change the whole file?
> 
> The picture with green circles and texts are there to attract the viewer's attention and show the steps clearly. I think it would be harder to "read" and understand for newcomers with only a mouse.
> 
> Also, I can redo the section of the video with the relay part. I don't see why we would remove the video. That being said, the relay change is not even yet on main net:
> 
> test_10
> 
> @mohamedamer453 I propose to let the file as it was before, and to adjust with the relay picture and change the small video portion when this is available for main net. (It is available for dev and test nets for now.)



> The [related issue](https://github.com/threefoldtech/info_grid/issues/51) asked for a small change of one picture to show that we need to create a relay instead of the twin: "The steps for creating a twin should be updated, to replace the twin ip creation part with selecting the relay."
> 
> Do we just change this picture or we change the whole file?
> 
> The picture with green circles and texts are there to attract the viewer's attention and show the steps clearly. I think it would be harder to "read" and understand for newcomers with only a mouse.
> 
> Also, I can redo the section of the video with the relay part. I don't see why we would remove the video. That being said, the relay change is not even yet on main net:
> 
> test_10
> 
> @mohamedamer453 I propose to let the file as it was before, and to adjust with the relay picture and change the small video portion when this is available for main net. (It is available for dev and test nets for now.)

I thought we can make the UI better by enhancing the screenshots and removing the duplicated title, there is no change on the file not related to the updated steps to create an account you can create one and check :D

In my opinion, the green circles are not the best thing for our UI, also there are some places there is no need for it the mouse is enough for example 
- in the whole page we have only one `add to chrome` button on the page, 
image
 is it really needed?
 
 I just want the docs to look simple and good for the user, I never see docs that make a circle on each step the user should do 

If you think that we need to keep the old pics, I will revert the changes of the img directory and only change the twin img.
please let me know your opinions @Mik-TF @mohamedamer453

**Comment by 0oM4R** [comment]
> Indeed, i totally agree with you replacing the mentioned image and adjusting other info about the twin ip/relay if needed was the main purpose of this issue, and i also totally agree about the video part if the section of the twin ip could be edited and replaced with the relay instead it would be much better than removing the whole video, since the video could be really helpful for new users. (and of course these changes should remain on the staging server and be released with the 3.9.0 mainnet release)

I add the video and the content table as in https://github.com/threefoldtech/info_grid/pull/57#discussion_r1149798160

**Comment by 0oM4R** [comment]
revert almost all the original images, will open a separate issue for screenshot enhancement, @Mik-TF can you check again?

**Review by Mik-TF — COMMENTED** [review]
The changes are good.

There's only some details needed for the TOC and overall presentation, based on @0oM4R's feedback.

Once this is done, on my side I would say that the pull request should be approved.

**Comment by Mik-TF** [comment]
> revert almost all the original images, will open a separate issue for screenshot enhancement, @Mik-TF can you check again?

I understand your point of view with how we present the information on screenshots. It would indeed be a good thing to have this in a separate issue.

File: src/getstarted/TF_Dashboard/TF_Dashboard.md [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
+
+# Threefold Dashboard: Create Account and Transfer TFT
```
Comment (Mik-TF) [inline src/getstarted/TF_Dashboard/TF_Dashboard.md:2]: Replace this line by this:

```
<h1> Threefold Dashboard: Create Account and Transfer TFT </h1>
```

File: src/getstarted/TF_Dashboard/TF_Dashboard.md [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
 
 <div class="youtubeVideoWrapper">
 <iframe title="Create an Account on ThreeFold Dashboard" width="560" height="315" src="https://www.youtube-nocookie.com/embed/t9rAZAPAWbM" frameborder="0" allowfullscreen="" sandbox="allow-same-origin allow-scripts allow-popups"></iframe>
 </div>
 
-<h2>Table of Contents</h2>
+## Table of Contents
```
Comment (Mik-TF) [inline src/getstarted/TF_Dashboard/TF_Dashboard.md:8]: Replace this line by this line

```
<h2> Table of Contents </h2>
```

File: src/getstarted/TF_Dashboard/TF_Dashboard.md [hunk 4]
```diff
@@ -1,18 +1,18 @@
...
 
 - [Threefold Dashboard: Create Account and Transfer TFT](#threefold-dashboard-create-account-and-transfer-tft)
```
Comment (Mik-TF) [inline src/getstarted/TF_Dashboard/TF_Dashboard.md:10]: Replace this TOC by this one:

```
- [Introduction](#introduction) 
- [Create Polkadot Extension Account](#create-polkadot-extension-account) 
- [Transfer TFT from Stellar Chain to TFChain](#transfer-tft-from-stellar-chain-to-tfchain)
```

File: src/getstarted/TF_Dashboard/TF_Dashboard.md [hunk 5]
```diff
@@ -1,18 +1,18 @@
...
+  - [Introduction](#introduction)
   - [Create Polkadot Extension Account](#create-polkadot-extension-account)
   - [Transfer TFT from Stellar Chain to TFChain](#transfer-tft-from-stellar-chain-to-tfchain)
 
-***
-
-# Threefold Dashboard: Create Account and Transfer TFT
+## Introduction
```
Comment (Mik-TF) [inline src/getstarted/TF_Dashboard/TF_Dashboard.md:15]: Replace this line by this line:

```
# Introduction
```

File: src/getstarted/TF_Dashboard/TF_Dashboard.md [hunk 6]
```diff
@@ -25,7 +25,7 @@ It is simple. The main steps are:
 
 ## Create Polkadot Extension Account
```
Comment (Mik-TF) [inline src/getstarted/TF_Dashboard/TF_Dashboard.md:26]: Replace this line by this line

```
# Create Polkadot Extension Account
```

**Review by Mik-TF — COMMENTED** [review]
The changes I wrote should work now.

Otherwise, having things like ## Table of Contents, before a table of contents, will have the unwanted effect of putting the line Table of Contents WITHIN the Table of Contents, if a future dev team member updates the Table of Contents, in VS Code or else.

**Comment by Mik-TF** [comment]
@0oM4R  Let me know if this is clear. First time I am reviewing stuff. Maybe I could have done this in a different way.

The changes needed are really just about the TOC. It's all here:

```
<h1> Threefold Dashboard: Create Account and Transfer TFT </h1>

<div class="youtubeVideoWrapper">
<iframe title="Create an Account on ThreeFold Dashboard" width="560" height="315" src="https://www.youtube-nocookie.com/embed/t9rAZAPAWbM" frameborder="0" allowfullscreen="" sandbox="allow-same-origin allow-scripts allow-popups"></iframe>
</div>

<h2> Table of Contents </h2>

- [Introduction](#introduction)
- [Create Polkadot Extension Account](#create-polkadot-extension-account)
- [Transfer TFT from Stellar Chain to TFChain](#transfer-tft-from-stellar-chain-to-tfchain)

# Introduction

[texts]

# Create Polkadot Extension Account

[texts]

# Transfer TFT from Stellar Chain to TFChain
```

**Comment by 0oM4R** [comment]
> The changes I wrote should work now.
> 
> Otherwise, having things like ## Table of Contents, before a table of contents, will have the unwanted effect of putting the line Table of Contents WITHIN the Table of Contents, if a future dev team member updates the Table of Contents, in VS Code or else.

This only happened if you have some markdown extensions, we can avoid this, 
my idea is to have only one 1 heading, `<h1>` or `#`, on the page, as the page has only one title.
so I think we need to  avoid setting the sections' titles with the first heading 
we can compare the overall looking of the page 
- having the three h1 titles
  image

- set the normal flow of the heading 
  image

**Comment by Mik-TF** [comment]
@0oM4R 

"This only happened if you have some markdown extensions, we can avoid this"

If we have tools that make us write guides quicker, we shouldn't "avoid" them.
Not using "update TOC" for long guides, is the opposite of being efficient.
And having to "adjust" an updated TOC isn't efficient either.

It doesn't matter if we use the two systems: `<h1>` and `#`, etc. There is no downsides to this. Not with HTML5.

"And especially with HTML5, having multiple H1 elements on a page is completely normal and kind of expected. So it's not something that you need to worry about. " John Muller, SEO talk for Google.

It also doesn't affect negatively SEO.

It's easier to write as we can generate TOC that are well organized. And it's easier for readers to read h1, h2. Starting with h3 and downward, the writing is almost as big as the normal text. So removing the option to use h1 in the texts isn't optimal for readability.

Check this example:

First line is h2, second h3, and third line is the normal text:

Screen Shot 2023-03-30 at 12 54 04 PM

Having many h1 won't affect SEO or anything. It's just way easier for the writer of the guide to be able to use quick tools like "Update table of contents".

So in short: using the two systems `<h1>` and `#`, as I proposed, it's faster to write guides, and it's easier for readers to discern texts and headings the very moment you have more than 2 levels of heading. And there's no downside with HTML5 for SEO, etc.

***

That being said, if you guys (everyone involved in the info_grid's manual) want to go forward with @0oM4R's changes, I won't stop anyone.

I made my point here and I think the arguments are solid.

Thanks @0oM4R  for your time in this pull request. Clearly I am picky with details but I think we both just want the best documentation for Threefold.

**Comment by Mik-TF** [comment]
@0oM4R 
What about this compromise:

We only have 1 heading level 1 (`<h1>`, or `#`) in there, but we do use the two systems (`<h1>` and also `#`, etc.), so the TOC can be generated automatically?

So we would have this:


```
<h1> Threefold Dashboard: Create Account and Transfer TFT </h1>

<div class="youtubeVideoWrapper">
<iframe title="Create an Account on ThreeFold Dashboard" width="560" height="315" src="https://www.youtube-nocookie.com/embed/t9rAZAPAWbM" frameborder="0" allowfullscreen="" sandbox="allow-same-origin allow-scripts allow-popups"></iframe>
</div>

<h2> Table of Contents </h2>

- [Introduction](#introduction)
- [Create Polkadot Extension Account](#create-polkadot-extension-account)
- [Transfer TFT from Stellar Chain to TFChain](#transfer-tft-from-stellar-chain-to-tfchain)

## Introduction

For this section, we will create an account on the TFChain and transfer TFT from Stellar chain to TFChain. We will then be able to use the TFT and deploy workloads on the Threefold Playground.

It is simple. The main steps are:

* Go to the Threefold Dashboard
* Download the Polkadot extension
* Create an account
* Send TFT from Stellar Chain to TFChain

## Create Polkadot Extension Account

...

## Transfer TFT from Stellar Chain to TFChain
```
What do you think? Let me know!

**Review by Mik-TF — APPROVED** [review]
